readthedocs/core/utils/__init__.py

```diff
@@ -134,3 +138,18 @@ def send_email(recipient, subject, template, template_html, context=None,
     context['uri'] = '{scheme}://{host}'.format(
         scheme='https', host=settings.PRODUCTION_DOMAIN)
     send_email_task.delay(recipient, subject, template, template_html, context)
+
+
+def slugify(value, *args, **kwargs):
+    """Add a DNS safe option to slugify
+
+    :param dns_safe: Remove underscores from slug as well
+    """
+    dns_safe = kwargs.pop('dns_safe', False)
```

Comment:
Feels like this should default to true. I generally think slugs shouldn't have _ and - in them, so happy to default it on.